ciao/cmd/show.go

```diff
@@ -0,0 +1,202 @@
+// Copyright © 2017 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/ciao-project/ciao/ciao/tool"
+	"github.com/spf13/cobra"
+)
+
+var showCmd = &cobra.Command{
```

Comment:
I don't like the overloading of show to mean show one and show multiple. Instead I think that should be a list command.